.github/CONTRIBUTING.md

```diff
@@ -32,15 +32,48 @@ traefik*
 
 - You need `go` v1.8+
 - It is recommended you clone Træfik into a directory like `~/go/src/github.com/containous/traefik` (This is the official golang workspace hierarchy, and will allow dependencies to resolve properly)
-- This will allow your `GOPATH` and `PATH` variable to be set to `~/go` via:
+- Set your `GOPATH` and `PATH` variable to be set to `~/go` via:
```

Comment:
I found that I still need to set the path manually other wise I get `generate.go:7: running "go-bindata": exec: "go-bindata": executable file not found in $PATH`:

```
jsturtevant:traefik (master)$ go get github.com/jteeuwen/go-bindata/...
jsturtevant:traefik (master)$ go generate
generate.go:7: running "go-bindata": exec: "go-bindata": executable file not found in $PATH
jsturtevant:traefik (master)$ go version
go version go1.8.3 linux/amd64
```
adding the two exports fix it.  Maybe I have something configured wrong?